tests/test_enterprise/test_signals.py

```diff
@@ -246,11 +248,213 @@ def test_default_content_filter(self, default_content_filter, expected_content_f
             assert enterprise_catalog.content_filter == expected_content_filter
 
 
+@mark.django_db
+class TestPendingEnterpriseAdminUserSignals(unittest.TestCase):
+    """
+    Test signals associated with PendingEnterpriseCustomerAdminUser.
+    """
+
+    def setUp(self):
+        """
+        Setup for `TestPendingEnterpriseAdminUserSignals` test.
+        """
+        self.admin_user = UserFactory(id=2, email='user@example.com')
+        self.enterprise_customer = EnterpriseCustomerFactory()
+        super(TestPendingEnterpriseAdminUserSignals, self).setUp()
+
+    def test_create_pending_enterprise_admin_user(self):
```

Comment:
nit: small thing, but I feel like the test naming (or the addition of a docstring) could make it a little more explicit that we're testing that creating the `PendingEnterpriseCustomerAdminUser` results in a non-admin customer user also being created

Reply:
Yup, missed adding the docstring here 😄 Test name reflects the name of the signal its testing.